service/src/test/java/uk/gov/hmcts/reform/fpl/service/DirectionHelperServiceTest.java

```diff
@@ -29,151 +39,706 @@
 
 @ExtendWith(SpringExtension.class)
 class DirectionHelperServiceTest {
-
     private final DirectionHelperService service = new DirectionHelperService();
 
-    @Test
-    void combineAllDirections_shouldAddRoleDirectionsIntoOneList() {
-        CaseData caseData = populateCaseDataWithFixedDirections()
-            .allPartiesCustom(buildCustomDirections())
-            .localAuthorityDirectionsCustom(buildCustomDirections())
-            .parentsAndRespondentsCustom(buildCustomDirections())
-            .cafcassDirectionsCustom(buildCustomDirections())
-            .otherPartiesDirectionsCustom(buildCustomDirections())
-            .courtDirectionsCustom(buildCustomDirections())
-            .build();
+    @Nested
+    class CombineAllDirectionsTest {
```

Comment:
If we need to do this maybe it would be worth having separate file? It's a bit difficult to review now, due to all file is marked as changed. Maybe could be refactored as separate PR?

Reply:
I have reverted lots of the changes, as I say, lots of new tests though so I am not sure how helpful it will be 😅